### PR TITLE
DOC-1896: RC: Update Redis Cloud docs for S3 backup that SSE-KMS is supported and move to bucket policy

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: 
https://docs.redis.com/staging/DOC-1896/rc/databases/back-up-data/#aws-s3
https://docs.redis.com/staging/DOC-1896/rc/databases/import-data/#via-aws-s3

### CODE REVIEW

**Review by banker — APPROVED** [review]
LGTM!

**Review by banker — APPROVED** [review]

File: content/rc/databases/back-up-data.md [hunk 1]
```diff
@@ -85,19 +85,84 @@ To store backups in an Amazon Web Services (AWS) Simple Storage Service (S3) [bu
 
 1. In the [AWS Management Console](https://console.aws.amazon.com/), use the **Services** menu to locate and select **Storage** > **S3**.  This takes you to the Amazon S3 admin panel.
 
-1.  Use the Buckets list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Access control list (ACL)** section, and then select the **Edit** button.
+1.  Use the Buckets list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Bucket policy** section, and then click **Edit**.
```
Comment (rrelledge) [inline content/rc/databases/back-up-data.md:88]:
```suggestion
1.  Use the **Buckets** list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Bucket policy** section, and then click **Edit**.
```

File: content/rc/databases/import-data.md [hunk 2]
```diff
@@ -69,17 +69,84 @@ To use the Redis Cloud admin console to import your data, you must first share t
...
-    1. In the Grantee field, enter:
-    `fd1b05415aa5ea3a310265ddb13b156c7c76260dbc87e037a8fc290c3c86b614`
-    1. In the Read column, select Yes.
-    1. Select Save.
-
-    For more info, see [Configuring ACLs for objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/managing-acls.html).
+1. In the [AWS Management Console](https://console.aws.amazon.com/), configure the bucket's [bucket policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-policies.html) to give access to Redis Cloud:
+    1. Use the **Services** menu to locate and select **Storage** > **S3**.  This takes you to the Amazon S3 admin panel.
+
+    1.  Use the Buckets list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Bucket policy** section, and then click **Edit**.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:75]:
```suggestion
    1.  Use the **Buckets** list to locate and select your bucket.  When the settings appear, select the **Permissions** tab, locate the **Bucket policy** section, and then click **Edit**.
```

File: content/rc/databases/import-data.md [hunk 3]
```diff
@@ -69,17 +69,84 @@ To use the Redis Cloud admin console to import your data, you must first share t
...
+
+        -  If there is no existing bucket policy, add the following JSON bucket policy. Replace `UNIQUE-BUCKET-NAME` with the name of your bucket.
+
+        {{% expand "CompetePolicy.json" %}}
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:79]: There is a rendering issue in this JSON example.

**Pull request merged**